misk/src/main/kotlin/misk/web/BoundAction.kt

```diff
@@ -115,8 +116,15 @@ internal class BoundAction<A : WebAction>(
       seedDataTransformers.fold(initialSeedData) { seedData, interceptor ->
         interceptor.transform(seedData)
       }
-    scope.enter(seedData).use {
-      handle(httpCall, pathMatcher)
+    
+    MDC.clear() // MDC should already be empty, but clear it again just in case
```

Comment:
I wonder if it's worth adding a test to validate the clearing is happening.

Reply:
Done!